site/content/how-to/control-plane-configuration.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: "Control Plane Configuration"
-description: "Learn how to dynamically update the Gateway Fabric control plane configuration."
-weight: 100
+title: "Control plane configuration"
+weight: 300
 toc: true
 docs: "DOCS-1416"
 ---
 
+Learn how to dynamically update the Gateway Fabric control plane configuration.
```

Comment:
I see what you're doing w/r/t moving descriptions to the start of a page -- have we set that up in one of our style guides?

Reply:
This isn't in a style guide: this is a workaround for a bug, not a desired long-term state.